models/enterprise_sonic/lldp_global/merged_example_01.txt

```diff
@@ -0,0 +1,35 @@
+# Using Merged
+#
+# Before State:
+# -------------
+#
+# sonic# show running-configuration | grep lldp
+# sonic#
+
+  - name: Modify LLDP configurations
+    dellemc.enterprise_sonic.sonic_lldp_global:
+      config:
+              enable: false
+              multiplier: 9
+              system_name : CR_sonic
+              hello_time: 18
+              mode: receive
+              system_description: Sonic_System
+              suppress_tlv_select: {
```

Comment:
Sorry I missed this earlier:

- The name that you have given this item in the model is "tlv_select" (not "suppress_tlv_select"). Please fix that here.
- Delete the open and close braces and the quotes around the attribute names. This is not correct for multi-line YAML dict definitions. (REF: https://docs.ansible.com/ansible/latest/reference_appendices/YAMLSyntax.html).

Reply:
addressed.